src/app/integrations/list/list.component.html

```diff
@@ -93,60 +93,44 @@ <h4 class="modal-title pull-left">Confirm {{ getActionTitle() }}</h4>
 
         <!-- Active/Inactive Labels -->
 
-        <div class="col-xs-5 active-inactive">
-          <div class="list-view-pf-additional-info-item">
-
-            <!-- In Progress -->
-
-            <div class="in-progress"
-                 *ngIf="integration.currentStatus === 'Pending'">
-              <h3>
-                <div class="spinner spinner-sm spinner-inline"></div>
-                In Progress
-              </h3>
-            </div>
-
-            <div *ngIf="integration.currentStatus === 'Activated' || integration.currentStatus === 'Deactivated'"
-                 (click)="toggleActive(integration)">
-              <label class="switch_wrap"
-                     for="field-{{ integration.id }}">
-                <input id="field-{{ integration.id }}"
-                       type="checkbox"
-                       [checked]="integration.currentStatus === 'Activated'" />
-                <div class="switch">
-                  <span class="bullet"></span>
-                </div>
-              </label>
-            </div>
+        <div class="col-xs-6 active-inactive">
+          <!-- In Progress -->
+
+          <div class="in-progress"
+               *ngIf="integration.currentStatus === 'Pending'">
+            <h3>
+              <div class="spinner spinner-sm spinner-inline"></div>
+              In Progress
+            </h3>
+          </div>
 
-            <div *ngIf="integration.currentStatus === 'Draft'"
-                 class="draft">
-              <span class="label label-warning" disabled>Draft</span>
-            </div>
+          <!-- Status -->
 
-            <div *ngIf="integration.currentStatus === 'Deleted'"
-                 class="deleted">
-              <span class="label label-danger" disabled>Deleted</span>
-            </div>
+          <div *ngIf="integration.currentStatus !== 'Pending'"
+               class="not-pending">
+              <span class="label label-{{ getLabelClass(integration.currentStatus) }}"
+                    disabled>{{ integration.currentStatus }}</span>
```

Comment:
We need to show 'Active' or 'Inactive', but using this unfortunately will mean we'll show 'Activated' and 'Deactivated' which is inconsistent with the wording on the dashboard.

Only reason I mention this is because I fixed this issue -> https://github.com/redhat-ipaas/ipaas-ui/issues/487